axolotlsay-js/dist.js

```diff
@@ -0,0 +1,30 @@
+const { bin } = require("./package.json");
+const execSync = require('child_process').execSync;
+
+// Compute the target we're building for
+const bunTargets = {
+    "x86_64-pc-windows-msvc": "bun-windows-x64",
+    "aarch64-apple-darwin": "bun-darwin-arm64",
+    "x86_64-apple-darwin": "bun-darwin-x64",
+    "aarch64-unknown-linux-gnu": "bun-linux-arm64",
+    "x86_64-unknown-linux-gnu": "bun-linux-x64"
+}
+const distTarget = process.env.CARGO_DIST_TARGET || process.env.DIST_TARGET;
+if (!distTarget) {
+    throw "DIST_TARGET isn't set, so we don't know what platform to build!"
```

Comment:
The original `build.sh` script was written assuming you might run it manually, but honestly I'm happier assuming this is cargo-dist only.